windows/src/test/unit-tests/kmcomp/test_casedkeys.kmn

```diff
@@ -0,0 +1,39 @@
+c Tests cased keys rule expansions
+store(&NAME) 'Test &CasedKeys'
+begin Unicode > use(main)
+
+group(main) using keys
+
+store(&CasedKeys) [K_A] [K_E] [K_I] [K_O] [K_U] [K_1] .. [K_4]
+
++ [K_A] > 'aaah'
++ [SHIFT K_A] > 'AAAH'
+
++ [K_E] > 'eh'
++ [SHIFT K_E] > 'EH'
+
++ [K_O] > 'oh'
++ [SHIFT K_O] > 'OH'
+
++ [K_1] > 'small one'
++ [SHIFT K_1] > 'big one'
```

Comment:
Oh, we're doing 'caps' on number keys, too?  That _was_ a question I had.

It's a bit odd b/c Caps Lock doesn't usually affect number keys... but it _is_ a reasonable option to provide, I suppose.

Reply:
Caps Lock affects keys in other locations on other keyboards, such as French AZERTY. See https://github.com/keymanapp/keyman/issues/2241#issuecomment-790384328 by @MattGyverLee for all the gory details!